openwisp_monitoring/db/backends/influxdb/client.py

```diff
@@ -117,13 +118,16 @@ def write(self, name, values, **kwargs):
         if isinstance(timestamp, datetime):
             timestamp = timestamp.isoformat(sep='T', timespec='microseconds')
         point['time'] = timestamp
-        self.get_db.write(
-            {'points': [point]},
-            {
-                'db': kwargs.get('database') or self.db_name,
-                'rp': kwargs.get('retention_policy'),
-            },
-        )
+        try:
+            self.get_db.write(
+                {'points': [point]},
+                {
+                    'db': kwargs.get('database') or self.db_name,
+                    'rp': kwargs.get('retention_policy'),
+                },
+            )
+        except Exception:
+            raise TimeseriesWriteException
```

Comment:
what about:

```python
except Exception as exception:
    logger.warning(f'got exception while writing to tsdb: {exception}')
    raise TimeseriesWriteException(exception)
```

This way we should see what's going on in the logs, please try it and adjust it so that we can clearly see what's wrong from the logs.